apidoc/ostree-sections.txt

```diff
@@ -246,6 +246,7 @@ ostree_repo_remote_get_gpg_verify
 ostree_repo_remote_get_gpg_verify_summary
 ostree_repo_remote_gpg_import
 ostree_repo_remote_fetch_summary
+ostree_repo_remote_fetch_summary_with_options
```

Comment:
You'll also need to add this to the `libostree.sym` file, and in fact since this will be the first symbol since 2016.5, you'll need to introduce that version.